JenkinsFile_build_container.groovy

```diff
@@ -40,7 +40,9 @@ timeout(time: 6, unit: 'HOURS') {
                         // Tag Regular build Containers with BUILD_NUMBER and 'latest'
                         TAGS = "-t ${NAMESPACE}/${CONTAINER_NAME}:${BUILD_NUMBER} -t ${NAMESPACE}/${CONTAINER_NAME}:latest"
                     }
-                    sh "docker build -f Dockerfile ${TAGS} ."
+                    dir('buildenv') {
+						sh "docker build -f Dockerfile ${TAGS} ."
+						}
```

Comment:
The change looks good. Just some formatting differences. It looks like you've used tabs on line 44,45. The rest of the file uses spaces. Switching back should resolve the appearance above.